src/main/java/ch/njol/skript/expressions/ExprChestInventory.java

```diff
@@ -37,65 +37,68 @@
 @Name("Custom Chest Inventory")
 @Description("Returns a chest inventory with the given amount of rows and the name. Use the <a href=effects.html#EffOpenInventory>open inventory</a> effect to open it.")
 @Examples({"open chest inventory with 1 row named \"test\" to player",
-           "set {_inventory} to chest inventory with 1 row"})
+	"set {_inventory} to chest inventory with 1 row"})
 @Since("2.2-dev34")
 public class ExprChestInventory extends SimpleExpression<Inventory> {
 
-    static {
-        Skript.registerExpression(ExprChestInventory.class, Inventory.class, ExpressionType.COMBINED,
-        		"[a [new]] chest inventory (named|with name) %string% [with %-number% row[s]]",
-        		"[a [new]] chest inventory with %number% row[s] [(named|with name) %-string%]");
-    }
+	static {
+		Skript.registerExpression(ExprChestInventory.class, Inventory.class, ExpressionType.COMBINED,
+			"[a [new]] chest inventory (named|with name) %string% [with %-number% row[s]]",
+			"[a [new]] chest inventory with %number% row[s] [(named|with name) %-string%]");
+	}
 
-    @Nullable
-    private Expression<Number> rows;
-    @Nullable
-    private Expression<String> name;
+	private static final String DEFAULT_CHEST_TITLE = "Chest";
+	private static final int DEFAULT_CHEST_ROWS = 3;
```

Comment:
You should probably replace these with InventoryType calls

so like InventoryType.CHEST.getDefaultTitle()